controllers/upload.js

```diff
@@ -0,0 +1,46 @@
+const Jimp = require("jimp");
+const path = require("path");
+const { updateAvatar } = require("../service/user.service");
+const { error } = require("console");
+const fs = require("fs").promises;
+
+const userUpdateAvatar = async ( req, res, next ) => {
+    const { path: tempPath, filename } = req.file;
```

Comment:
Tutaj sprawdź sobie czy req.file jest i czy ma wszystkie parametry. Jeśli nie to zkończ wykonywanie skryptu.